django/views/decorators/cache.py

```diff
@@ -5,8 +5,10 @@
 from django.utils.cache import add_never_cache_headers, patch_cache_control
 from django.utils.decorators import decorator_from_middleware_with_args
 
+_sentinel = object()
 
-def cache_page(timeout, *, cache=None, key_prefix=None):
+
+def cache_page(timeout, *, cache=None, key_prefix=_sentinel):
```

Comment:
I don't think this sentinel way is a good solution, but I don't know a better way to do that, because `None` is a valid config to pass to `CacheMiddleware`.